deploy/snapshot/cmd/cuda-vmm-interpose/test/run.sh

```
@@ -0,0 +1,161 @@
+root="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+build="${BUILD_DIR:-$(mktemp -d)}"
+trap 'rm -rf "$build"' EXIT
+
+sed "s|/usr/local/lib/dynamo/libdynamo_snapshot_cuda_vmm.so|/lib/x86_64-linux-gnu/libm.so.6|" \
+  "$root/launch.sh" >"$build/launch"
+chmod +x "$build/launch"
+launcher_output="$(
+  DYN_SNAPSHOT_JOB_ID=11111111111111111111111111111111 \
+  LD_PRELOAD=/lib/x86_64-linux-gnu/libm.so.6 \
+    "$build/launch" /bin/sh -c \
+      'printf "%s|%s|%s" "$DYN_SNAPSHOT_CUDA_VMM_INTERPOSE" "$LD_PRELOAD" "$1"' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)